src/style.css

```diff
@@ -1,3 +1,62 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+header {
+  height: 60px;
+  width: 100%;
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+}
+
+.links {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 20px;
+}
+
+.link {
+  text-decoration: none;
+  color: rgb(0, 0, 0);
+  position: relative;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  font-weight: 400;
+}
+
+.imagem {
```

Comment:
Don't use tag names for styling (except 'html' and 'body'). Style all elements using '.class' and if needed with ':pseudo-class', 'pseudo-element' and '[attribute]'.